playbooks/restore_backup.yml

```diff
@@ -4,7 +4,7 @@
   remote_user: "{{ unicorn_user }}"
 
   tasks:
-    - name: restore backup
+    - name: restore backup # noqa 301
```

Comment:
What does this comment mean?

Reply:
As noted in the PR description, it's the preferred syntax for disabling specific lint failures: https://github.com/ansible/ansible-lint#false-positives-skipping-rules